app/components/TaskList/index.tsx

```diff
@@ -263,14 +266,10 @@ const TaskList: React.FC<Props> = ({
           </TaskListSection>
         )}
         {/* Amendments & Other Revisions section */}
-        {useShowGrowthbookFeature("amendments") && mode === "view" && (
+        {useShowGrowthbookFeature("amendments") && mode !== "create" && (
           <ProjectRevisionChangeLogsTaskListSection
             projectRevisionId={id}
-            defaultExpandedState={[
-              getProjectRevisionChangeLogsPageRoute(id).pathname,
-              projectRevisionViewPagePathName,
-              projectRevisionCreatePagePathName,
-            ].includes(router.pathname)}
+            defaultExpandedState={true}
```

Comment:
There's already conditional logic in TaskListItem to hide the subheading and caret when we don't want to see it, so this can always be expanded